zh_cn.lang

```diff
@@ -151,198 +151,198 @@ gui.xaero_world_server=世界/服务器
-gui.xaero_light_overlay_type=光照叠加
+gui.xaero_entity_radar_type=类型：
+gui.xaero_entity_radar_color=颜色：
+gui.xaero_entity_radar_visiblity=可见性：
+gui.xaero_entity_radar_names=名称：
+gui.xaero_entity_radar_icons=图标/头像：
+gui.xaero_box_entity_radar_names=在点或图标/头像下渲染实体名称。 \n \n 选项“玩家列表” - 仅在按下“玩家列表”键（默认为 TAB）时显示。
+gui.xaero_light_overlay_status=使用光照叠加层
+gui.xaero_light_overlay=光照叠加层
+gui.xaero_light_overlay_type=光照叠加层
 gui.xaero_light_overlay_color=颜色
-gui.xaero_light_overlay_max_light=最大亮度
-gui.xaero_light_overlay_min_light=最小亮度
-gui.xaero_toggle_light_overlay=光照叠加开关
-gui.xaero_box_light_overlay_type=启用光照叠加会降低游戏性能，因为目前未使用来自World Map模组的区块数据，此功能会增加小地图刷新频率。请不要一直开启此功能。
-gui.xaero_day=Day
-gui.xaero_tamed=驯养的动物
-gui.xaero_untamed_color=未驯养的动物颜色
-gui.xaero_icons_list=Player List
+gui.xaero_light_overlay_max_light=最大光照等级
+gui.xaero_light_overlay_min_light=最小光照等级
+gui.xaero_toggle_light_overlay=切换光照叠加层
+gui.xaero_box_light_overlay_type=使用此叠加层会对游戏性能产生负面影响， 因为它目前不使用世界地图模组区块并增加小地图更新频率。 不要一直保持在开启状态。
+gui.xaero_day=天
+gui.xaero_tamed=驯服的生物
+gui.xaero_untamed_color=未驯服颜色
+gui.xaero_icons_list=玩家列表
 gui.xaero_icons_always=总是
-gui.xaero_icons_off=无头像
-gui.xaero_box_entity_radar_icons=渲染实体的图标而不是彩色点。\n选项“按下玩家列表键时” - 仅当“玩家列表”按键（默认为TAB）按下时才渲染头像。
-gui.xaero_dot_name_scale=点名称尺寸
-gui.xaero_entity_heads_scale=实体图标尺寸
-gui.xaero_box_minimap_size=设置小地图的尺寸。当尺寸设置大于180时方形小地图会强制进入“锁定南北”模式。 \n \n 使用“自动”选项请滑至最左侧。 \n \n 旧版本尺寸所对应的值： \n 极小 - 57 \n 小 - 85 \n 中 - 113 \n 大 - 169
+gui.xaero_icons_off=无图标
+gui.xaero_box_entity_radar_icons=渲染实体图标而不是彩色点。 \n \n 选项“玩家列表” - 仅在按下“玩家列表”键（默认为 TAB）时显示。
+gui.xaero_dot_name_scale=点名称缩放
+gui.xaero_entity_heads_scale=实体图标缩放
+gui.xaero_box_minimap_size=影响可见程度的小地图框体大小。 当大小大于180时，方形小地图会强制进入非旋转模式（选项“锁定小地图北方”）。 \n \n 对于“自动”选项，一直拉到左边。 \n \n 旧版本模组中的大小值： \n 微小 - 57 \n 小 - 85 \n 中 - 113 \n 大 - 169
 gui.xaero_auto_clamp_depth=自动
-gui.xaero_waypoints_clamp_depth=距离阈值
-gui.xaero_waypoints_distance_scale=路径点距离文本尺寸
-gui.xaero_waypoints_name_scale=路径点名称文本尺寸
-gui.xaero_box_waypoints_clamp_depth=当你在游戏中远离路径点超过此距离后，路径点将不再缩小。\n 若你想调整远处的路径点的尺寸时，可以调整此值。使用“游戏内路径点尺寸”设置来调整你靠近路径点时的尺寸。 \n \n 将滑块拉到最左边可以启用“自动”选项。\n 自动选项将会根据你的角视场、屏幕尺寸和默认的路径点尺寸来计算一个最优值。\n \n 旧版模组的默认值： \n 距离阈值
-gui.xaero_box_waypoints_distance_scale=游戏内路径点距离显示文本的尺寸。 \n \n 旧版模组的默认值： \n 距离阈值
-gui.xaero_box_waypoints_name_scale=游戏内路径点名称显示文本的尺寸。 \n \n 旧版模组的默认值： \n 距离阈值
-gui.xaero_box_waypoints_scale=游戏内路径点尺寸。使用此值来改变所有游戏内路径点尺寸，包括那些离你很近的路径点。使用 \距离阈值\ 设置来指定较远的路径点的显示大小。\n \n 旧版模组的默认值： \n 距离阈值
-gui.xaero_waypoint_shared=%1$s 分享了一个名为 \%2$s\ 的路径点！§2§n[添加]
-gui.xaero_waypoint_shared_dimension=%1$s 在 %3$s 分享了一个名为 \%2$s\ 的路径点！§2§n[添加]
+gui.xaero_waypoints_clamp_depth=深度限制
+gui.xaero_waypoints_distance_scale=路径点距离文本缩放
+gui.xaero_waypoints_name_scale=路径点名称文本缩放
+gui.xaero_box_waypoints_clamp_depth=当你远离它们时，游戏内路径点停止变小的屏幕深度。 \n 如果你想在屏幕上使远处的路径点变大/变小，请使用此选项。使用“游戏内路径点缩放”设置也可以在你靠近它们时影响路径点的大小。 \n \n 对于“自动”选项，一直拉到到左边。 \n 自动选项会同时考虑你的 FOV 和屏幕高度，尝试为你提供最佳的缩放默认值，。 \n \n 旧模组版本中的默认值：\n 深度限制：12 \n 游戏内缩放：2.5 \n 名称文本缩放：0.5 \n 距离文本缩放：0.8
```

Comment:
Yeah? I found out that it still works when I tested the mod for 1.12.2

Reply:
These settings are not in the 1.12.2 version or newer. It uses a new rendering system.

Reply:
Since the option doesn't exist, I think there is no problem to translate the original EN text to "深度限制".

Reply:
The option exists. I still update 1.7.10 and 1.8.9 with bug fixes and translations.

Reply:
Ok, I will download 1.7.10 and check

Reply:
Checked, the translation still works for 1.7 and 1.8

Reply:
Are you talking about the tooltip? Because that's what I'm talking about.
When I translate it back, I get "In-game waypoints stop getting smaller when you go further away from them", with no mention of any depth. That's why I left the comment.

Reply:
I guess this is a problem with translation tools, because I did mention "depth" (深度).

Reply:
Awesome, then it seems correct!